main.go

```diff
@@ -158,6 +158,10 @@ func main() {
 				// Now, loop through the new list, which after the above deletions, contains only stuff that needs to
 				// have a new informer created for it.
 				for gvr := range gvrList {
+					fmt.Println("Found new resource %s, creating informer\n", gvr.String())
+					if gvr.Resource == "configmaps" {
+						fmt.Println("FOUND CONFIGMAP RESOURCE", gvr.String())
+					}
 					glog.V(2).Infof("Found new resource %s, creating informer\n", gvr.String())
```

Comment:
This path should work now.  When a new resource is allowed I see this log (after a few minutes) and the new resource gets collected.

The problem is when we remove a resource that's being watched.  We stop the informer, but don't remove the data. The informer gets stopped a few lines above here, in this block.
```go
glog.V(2).Infof("Resource %s no longer exists or no longer supports watch, stopping its informer\n", gvr.String())
close(stopper)
delete(stoppers, gvr)
```

**A better starting point is where the informer exits its execution, around this line**. https://github.com/stolostron/search-collector/blob/main/pkg/informer/informer.go#L51-L53

The informer needs to trigger a delete event for each resource in the `resourceIndex` before it terminates.